src/main/java/org/betonquest/betonquest/compatibility/holographicdisplays/HologramWrapper.java

```diff
@@ -0,0 +1,122 @@
+package org.betonquest.betonquest.compatibility.holographicdisplays;
+
+import lombok.CustomLog;
+import lombok.Getter;
+import me.filoghost.holographicdisplays.api.hologram.Hologram;
+import me.filoghost.holographicdisplays.api.hologram.VisibilitySettings;
+import org.betonquest.betonquest.BetonQuest;
+import org.betonquest.betonquest.api.config.QuestPackage;
+import org.betonquest.betonquest.compatibility.holographicdisplays.lines.AbstractLine;
+import org.betonquest.betonquest.id.ConditionID;
+import org.betonquest.betonquest.utils.PlayerConverter;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+import java.util.List;
+
+/**
+ * Wrapper class for {@link Hologram} that stores data parsed from hologram configuration inside <code>custom.yml</code>.
+ */
+@SuppressWarnings("PMD.CommentRequired")
```

Comment:
New Classes should not have any `@SuppressWarnings`, except false positives

And `CommentRequired` is not allowed at all for new Classes.